invenio_jsonschemas/ext.py

```diff
@@ -165,6 +167,17 @@ def path_to_url(self, path):
         ).build(
             'schema', values={'path': path}, force_external=True)
 
+    @cached_property
+    def loader(self):
+        """Loader used by `JsonRef.replace_refs`."""
+        try:
+            loader_ = import_string(self.app.config['JSONSCHEMAS_LOADER_CLS'])
+        except ImportError:
```

Comment:
Please do not catch the exception here otherwise it will be impossible to detect misconfiguration. 

Have a look at pattern for loading configuration we use in other packages e.g. https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/utils.py#L76-L87